ush/setup.sh

```diff
@@ -1125,11 +1125,14 @@ fi
 #
 # Set the full path to the experiment directory.  Then check if it already
 # exists and if so, deal with it as specified by PREEXISTING_DIR_METHOD.
+#  IF $NEW_EXPTDIR = FALSE, overwrite same files under $EXPTDIR
 #
 #-----------------------------------------------------------------------
 #
 EXPTDIR="${EXPT_BASEDIR}/${EXPT_SUBDIR}"
-check_for_preexist_dir_file "$EXPTDIR" "${PREEXISTING_DIR_METHOD}"
+if [ "${NEW_EXPTDIR}" = "TRUE" ]; then
+  check_for_preexist_dir_file "$EXPTDIR" "${PREEXISTING_DIR_METHOD}"
+fi
```

Comment:
This would be best as a new "method" for the `check_for_preexist_dir_file` function. Maybe `PREEXISTING_DIR_METHOD` could be "overwrite" and it does nothing. It gets a bit confusing when we have two separate variables that handle the same thing -- both `NEW_EXPTDIR` and `PREEXISTING_DIR_METHOD`, but `NEW_EXPTDIR` overrides anything set in  `PREEXISTING_DIR_METHOD`.

Reply:
Yeah, it will be great to add "overwrite" and "intact" options to "PREEXISTING_DIR_METHOD".  "intact" is used to keep $STMP intact. So the requirement for some workflow runs is to overwrite $EXPTDIR and keep $STMP intact. I did not dig deep into check_for_preexist_dir_file(..) yet. Could you or someone else implement it? Thanks!

Reply:
I do not have the cycles for this endeavor. This does break existing capabilities, so it would be great if you could address it before this PR is merged.